bors.toml

```diff
@@ -3,7 +3,7 @@ pr_status = [ "DCO" ]
 required_approvals = 2
 block_labels = [ "do not merge" ]
 delete_merged_branches = true
-timeout_sec = 7200 # two hours
+timeout_sec = 43200 # Twelve hours
```

Comment:
Can we add a quick comment above this line with a link to the bors issue? Otherwise someone may happen across this odd value and change it back.